foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -52,9 +52,15 @@
 
   private static final String MICROSERVICE_CONFIG_LOADER_KEY = "cse-microservice-config-loader";
 
+  private static ConfigModel model = new ConfigModel();
+
   private ConfigUtil() {
   }
 
+  public static void addConfigs(Map<String, Object> config) {
```

Comment:
As this method is just set the internal cofigModel,  it's better to use setConfigs instead of addConfigs.